app/views/verify/finance/new.html.slim

```diff
@@ -1,29 +1,34 @@
 - title t('idv.titles.session.finance')
 
 h1.h3.my0 = t('idv.titles.session.finance')
-p.mt-tiny.mb0 = t('idv.messages.finance.intro_ccn')
+p.mt-tiny.mb0
+  = t('idv.messages.finance.intro_ccn_html',
+      intro_help_link: link_to(t('idv.messages.finance.intro_ccn_help'), help_path))
+
 = simple_form_for(idv_finance_form, url: verify_finance_path,
     html: { autocomplete: 'off', method: 'put', role: 'form' }) do |f|
   = hidden_field_tag 'idv_finance_form[finance_type]', :ccn
-  .clearfix.mxn1
+
+  .clearfix.mxn1.px1
     - Idv::FinanceForm.ccn_inputs.each do |key, label, html_options|
-      .px1
+      div
         = f.label key, label: label, class: 'block bold'
         p.mb1.italic = t('idv.messages.finance.hint')
-      .col.col-12.sm-col-8.px1
+      .col.col-12.sm-col-8
```

Comment:
I think the idea here originally was that on mobile screen the submit button and input box could all be inline, but now since we have the extra text inbetween the input and button this doesn't need the `.col`